interfaces/mount/entry.go

```diff
@@ -168,3 +169,24 @@ func LoadFSTab(reader io.Reader) ([]Entry, error) {
 	}
 	return entries, nil
 }
+
+// SaveFSTab writes a list of entries to a fstab-like file.
+//
+// The supported format is described by fstab(5).
+//
+// Note that there is no support for comments, both the LoadFSTab function and
+// SaveFSTab just ignore them.
+//
+// Note that there is no attempt to use atomic file write/rename tricks. The
```

Comment:
Enhance your calm:
> C:   zyga— what's the expected use case for `SaveFSTab`?
> Z:      Chipaca: snap-update-ns will write the things it did to /run
> Z:      Chipaca: it will be a file in /run/snapd/ns/$SNAP_NAME.fstab
> C:   so, /run doesn't need atomicity in the sense we mean there
> Z:      Chipaca: it should match /var/lib/snapd/mount/
> C:   but /var does
> Z:      Chipaca: but if it crashes the code will pick up from the start
> Z:      Chipaca: and see if it needs to mount anything
> C:   ah, if snapd crashes? yeah
> Z:      Chipaca: not snapd, this will be in a separate process
> C:   k
> C:   i was thinking of the machine crashing, not the program crashing
> Z:      Chipaca: if machine crashes it's all good
> Z:      Chipaca: this is purely ephemeral
> C:   yes
> C:   zyga— in what sense should things in run match things in var?
> Z:      if all is good yes
> Z:      some changes can be rejected
> Z:      so they will not be mounted
> Z:      and such entries won't show up
> Z:      some entries can fail for various reasons
> Z:      this is just a helper for the algorithm we merged earlier
> Z:      that descries 'current' state
> Z:      the desired state is always in /var/
> Z:      written by snapd
> Z:      using all the nice atomic helpers
> Z:      and the current file will be only written by this thing
> Z:      and maybe by snap-confine but this is really optional since it will cross-check with mountinfo to see what is really mounted
> C:   zyga— ok. Can you add a note about this to the method?
> Z:      Chipaca: sure
> C:   zyga, thanks